### PR TITLE
feat: add sendAudioEvent and isVoiceMessage method to msc extensions

### PR DESCRIPTION
**Pull request by coder-with-a-bushido**
As td suggested here -> https://github.com/famedly/app/pull/5749#discussion_r1546030031

### CODE REVIEW

File: lib/src/room.dart [hunk 1]
```diff
@@ -653,6 +653,37 @@ class Room {
         threadLastEventId: threadLastEventId);
   }
 
+  /// Sends an audio file with appropriate info to this room. Returns the event
+  /// ID generated by the server for this event.
+  Future<String?> sendAudioEvent(
```
Comment (td-famedly) [inline lib/src/room.dart:658]: Should go into msc_extensions/ iirc until stable

**Review by td-famedly — DISMISSED** [review]

File: lib/msc_extensions/extension_audio_events/audio_events.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import 'package:matrix/matrix.dart';
+
+extension AudioEventExtension on Event {
+  /// returns if the event is a voice message
+  bool get isVoiceMessage =>
+      content['org.matrix.msc2516.voice'] != null ||
+      content['org.matrix.msc3245.voice'] != null;
```
Comment (krille-chan) [inline lib/msc_extensions/extension_audio_events/audio_events.dart:7]: Can we maybe instead check if this is a `Map<String,Object?>`? If someone puts in an `int` in this or something else, I think I would expect that this getter returns false. What do you think?

File: lib/src/room.dart (hunk 1 above)
Comment (td-famedly) [reply lib/src/room.dart]: makes sense

File: lib/msc_extensions/extension_audio_events/audio_events.dart (hunk 2 above)
Comment (coder-with-a-bushido) [reply lib/msc_extensions/extension_audio_events/audio_events.dart:7]: sure, sounds good to me.

**Review by krille-chan — APPROVED** [review]

**Pull request merged**